docs/web3-eth-txpool.rst

```diff
@@ -0,0 +1,265 @@
+            563669: ["0x3dcb4c90477a4b8ff7190b79b524773cbe3be661: 1047965690000000000 wei + 90000 × 20000000000 gas"],
+            563670: ["0x6dfef5bc94b031407ffe71ae8076ca0fbf190963: 1047859050000000000 wei + 90000 × 20000000000 gas"]
+            },
+            0x9174e688d7de157c5c0583df424eaab2676ac162: {
+            3: ["0xbb9bc244d798123fde783fcc1c72d3bb8c189413: 30000000000000000000 wei + 85000 × 21000000000 gas"]
+            },
+            0xb18f9d01323e150096650ab989cfecd39d757aec: {
+            777: ["0xcd79c72690750f079ae6ab6ccd7e7aedc03c7720: 0 wei + 1000000 × 20000000000 gas"]
+            },
+            0xb2916c870cf66967b6510b76c07e9d13a5d23514: {
+            2: ["0x576f25199d60982a8f31a8dff4da8acb982e6aba: 26000000000000000000 wei + 90000 × 20000000000 gas"]
+            },
+            0xbc0ca4f217e052753614d6b019948824d0d8688b: {
+            0: ["0x2910543af39aba0cd09dbb2d50200b3e800a63d2: 1000000000000000000 wei + 50000 × 1171602790622 gas"]
+            },
+            0xea674fdde714fd979de3edf0f56aa9716b898ec8: {
+            70148: ["0xe39c55ead9f997f7fa20ebe40fb4649943d7db66: 1000767667434026200 wei + 90000 × 20000000000 gas"]
+            }
+        },
+        queued: {
+            0x0f6000de1578619320aba5e392706b131fb1de6f: {
+            6: ["0x8383534d0bcd0186d326c993031311c0ac0d9b2d: 9000000000000000000 wei + 21000 × 20000000000 gas"]
+            },
+            0x5b30608c678e1ac464a8994c3b33e5cdf3497112: {
+            6: ["0x9773547e27f8303c87089dc42d9288aa2b9d8f06: 50000000000000000000 wei + 90000 × 50000000000 gas"]
+            },
+            0x976a3fc5d6f7d259ebfb4cc2ae75115475e9867c: {
+            3: ["0x346fb27de7e7370008f5da379f74dd49f5f2f80f: 140000000000000000 wei + 90000 × 20000000000 gas"]
+            },
+            0x9b11bf0459b0c4b2f87f8cebca4cfc26f294b63a: {
+            2: ["0x24a461f25ee6a318bdef7f33de634a67bb67ac9d: 17000000000000000000 wei + 90000 × 50000000000 gas"],
+            6: ["0x6368f3f8c2b42435d6c136757382e4a59436a681: 17990000000000000000 wei + 90000 × 20000000000 gas", "0x8db7b4e0ecb095fbd01dffa62010801296a9ac78: 16998950000000000000 wei + 90000 × 20000000000 gas"],
+            7: ["0x6368f3f8c2b42435d6c136757382e4a59436a681: 17900000000000000000 wei + 90000 × 20000000000 gas"]
+            }
+        }
+    }
+
+
+------------------------------------------------------------------------------
+
+
+status
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.txpool.status([, callback])
+
+This will provide the number of transactions currently pending for inclusion in the next block(s), as well as the ones that are being scheduled for future executions.
+The RPC method used is ``txpool_status``.
+
+----------
+Parameters
+----------
+
+
+1. ``Function`` - (optional) Optional callback, returns an error object as first parameter and the result as second.
+
+
+-------
+Returns
+-------
+
+
+``Object`` - A list of number of pending and queued transactions.
```

Comment:
Please dont just put object write the current interface it returns with full details in which field does what else users will just raise issues. Also if its a promise make sure you do Promise<Type> so people know that.

Reply:
okay.